src/main/java/com/celements/pagelayout/DefaultLayoutService.java

```diff
@@ -345,7 +345,11 @@ public final boolean canRenderLayout(SpaceReference layoutSpaceRef) {
           .filter(getPageLayoutPropertiesClassRef())
           .stream().findFirst();
     } catch (DocumentNotExistsException exp) {
-      LOGGER.info("Layout property doc [{}] does not exist.", layoutPropDocRef, exp);
+      if (LOGGER.isTraceEnabled()) {
+        LOGGER.trace("Layout property doc [{}] does not exist.", layoutPropDocRef, exp);

```

Comment:
what is the intent here? we never log exception stacktraces on trace level.

Reply:
It is not a bug if the LayoutProperty-Doc does not exist. Thus it can get very verbose if you get a stacktrace for ever getLayoutPropertyBaseObject call for  a non layout. Nevertheless, if you are debugging and not expecting a call with the given document reference, the stacktrace helps you to figure out what happens.